docs/installation/installation.md

```diff
@@ -96,6 +99,47 @@ kubectl get secret/tyk-operator-conf -n tyk-operator-system -o json | jq '.data'
 }
 ```
 
+#### Tyk Pro
+
+- If you installed Tyk Pro using Helm, by default `tyk-operator-conf` is created with the following keys; `TYK_AUTH`, `TYK_MODE`, `TYK_ORG`, and `TYK_URL`. 
+
+The following command shows how you can access the value of `TYK_ORG` assuming that Tyk Pro is installed in the `tyk` namespace:
+```bash
+kubectl get secrets -n tyk tyk-operator-conf --template={{.data.TYK_ORG}} | base64 -d
+```
+
+- If you installed Tyk Pro manually, then you can access `TYK_AUTH` and `TYK_ORG` values from the dashboard as well.
+
+Under the Users page, click on any user to find associated values for that particular user.
+
+![tyk-dashboard](../img/tyk-dashboard-values.png)
+
+`TYK_AUTH` corresponds to `Tyk Dashboard API Access Credentials` in the above image, indicated by `1`.
+
+`TYK_ORG` corresponds to `Organisation ID` in the above image, indicated by `2`.
+
+#### Tyk CE
+
+If Tyk CE is deployed, `tyk-operator-conf`, Kubernetes Secret object, should be created through kubectl.
+
+
+Suppose, Tyk CE is installed in the `tykce-control-plane` namespace through Helm based on [values.yaml](https://github.com/TykTechnologies/tyk-operator/blob/master/ci/helm/tyk-headless/values.yaml).
+- `TYK_AUTH` corresponds to the value of the `secrets.APISecret`,
+- `TYK_ORG` corresponds to the value of the `secrets.OrgID`,
```

Comment:
Not sure if we need to set TYK_ORG in CE mode. Since there is no concept of Organization in CE mode.

Reply:
Thank you @komalsukhani for spotting this! I updated the guideline.

Side note: I added TYK_ORG based on the values defined aforementioned values file for tyk-headless installation.

Reply:
Just checking - even in CE mode, api keys, policies and session tokens are tied to the org id. 

Is it a good idea to remove this config option?

Reply:
Ohh sorry I didn't knew that.
Thanks for saving @asoorm

Reply:
Hi I'd like to clarify the use of Org ID for CE mode. 

In the "tyk-headless" values.yaml from helm repo, there's no secrets.OrgID; so user may not know what to set in here. Is there a default org ID for CE mode?

Reply:
It's because of the API keys. if you change the ORG id, then the API key will stop working. It supports multi-tenancy. Worth double checking with @buger for the open source / headless use-case.